src/target/cortexm.c

```diff
@@ -1062,6 +1062,25 @@ static uint32_t cortexm_dwt_func(target_s *target, target_breakwatch_e type)
 	}
 }
 
+static uint32_t cortexm_dwt_func_v8m(target_s *target, target_breakwatch_e type, size_t len)
+{
+	uint32_t x = CORTEXM_DWT_FUNC_ACTION_DEBUG_EVENT_V8M;
+	switch (type) {
+	case TARGET_WATCH_WRITE:
+		x |= CORTEXM_DWT_FUNC_MATCH_WRITE_V8M;
+		break;
+	case TARGET_WATCH_READ:
+		x |= CORTEXM_DWT_FUNC_MATCH_READ_V8M;
+		break;
+	case TARGET_WATCH_ACCESS:
+		x |= CORTEXM_DWT_FUNC_MATCH_ACCESS_V8M;
+	default:
+		return -1;
```

Comment:
This does an implementation defined conversion, please use `UINT32_MAX` instead or a result like `0U` for less code generation.